pylint_odoo/checkers/modules_odoo.py

```diff
@@ -826,6 +826,8 @@ def _check_xml_attribute_translatable(self):
         """The xml attribute is missing the translation="off" tag
             Example  <attribute name="groups">sale.group</attribute>
         """
+        if self.config.valid_odoo_versions != ['8.0']:
```

Comment:
@moylop260 
I try to get self.config.valid_odoo_versions But gives an error AttributeError: Values instance has no attribute 'valid_odoo_versions' https://travis-ci.org/Vauxoo/pylint-odoo/jobs/200304075#L950